Lares/Assets/Mechanics/Inventory/InventoryItem.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using Unity.VisualScripting;
+using UnityEngine;
+
+namespace Lares.Inventory
+{
+    public interface IItem
+    {
+        public void OnUse();
+    }
+
+    [Serializable]
+    public class InventoryItem : IItem
+    {
+        [SerializeField] public InventoryData ItemData;
```

Comment:
Does not need to be serialized if it is already public.
```suggestion
        public InventoryData ItemData;
```